File: september/07-09/07-09.py
def parse_roman_numeral(numeral):
    
    romans = {"I": 1, "V": 5, "X": 10, "L": 50, "C": 100, "D": 500, "M": 1000}
    
    suma = 0
    
    for i in range(len(numeral)-1):
        if(romans[numeral[i]] < romans[numeral[i+1]]):
            suma -= romans[numeral[i]]
        else:
            suma += romans[numeral[i]]
    
    suma += romans[numeral[len(numeral)-1]]
    
    #print(suma)
    return suma

File: september/07-09/07-09.js
function parseRomanNumeral(numeral) {
  
    const romans = {I: 1, V: 5, X: 10, L: 50, C: 100, D: 500, M: 1000};

    let sum = 0;

    for (let index = 0; index < numeral.length -1; index++) {

        if (romans[numeral[index]] < romans[numeral[index +1]] ) {
            sum -= romans[numeral[index]];
        } else {
            sum += romans[numeral[index]];
        }
    }
  
  sum += romans[numeral[numeral.length -1]]

  //console.log(sum)
  return sum;
}